### PR TITLE
fix(route): the Bloomberg RSS doesn't work after using the new ofetch as the 'got' lib

### PR DESCRIPTION
**Pull request by bigfei**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/bloomberg
/bloomberg/green
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
By using the old/deprecated 'got' because the 'redirectUrls' feature is not supported in the new 'got' library.

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Deprecated got is only available in dev environment

**Comment by Urabartin** [comment]
> Deprecated got is only available in dev environment

New `got` is kind of confined. What if origin url after redirect is needed?
#15117

**Comment by bigfei** [comment]
@TonyRL Changed to use the raw method from `ofetch` so that the redirected prop from Response can be used.

File: lib/utils/got.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ const getFakeGot = (defaultOptions?: any) => {
             delete options.parseResponse;
         }
 
-        const response = ofetch(request, options);
+        const response = options?.rawResponse ? ofetch.raw(request, options) : ofetch(request, options);
```
Comment (TonyRL) [inline lib/utils/got.ts:56]: Don't touch `lib/utils/got.ts`, import '@/utils/ofetch' directly instead

**Comment by bigfei** [comment]
@TonyRL now use ofetch.raw directly in `utils.ts`, please help do the PR

File: lib/routes/bloomberg/index.ts [hunk 2]
```diff
@@ -16,9 +16,37 @@ const site_title_mapping = {
...
+    | bbiz      | Business   |
+    | markets   | Markets    |
+    | technology| Technology |
+    | green     | Green      |
+    | wealth    | Wealth     |
+    | pursuits  | Pursuits   |
+    | bview     | Opinion    |
+    | equality  | Equality   |
+    | businessweek | Businessweek |
+    | citylab   | CityLab    |
```
Comment (TonyRL) [inline lib/routes/bloomberg/index.ts:48]:
```suggestion
    | Site ID      | Title        |
    | ------------ | ------------ |
    | /            | News         |
    | bpol         | Politics     |
    | bbiz         | Business     |
    | markets      | Markets      |
    | technology   | Technology   |
    | green        | Green        |
    | wealth       | Wealth       |
    | pursuits     | Pursuits     |
    | bview        | Opinion      |
    | equality     | Equality     |
    | businessweek | Businessweek |
    | citylab      | CityLab      |
```

**Pull request merged**